java/bulk-export/bulk-export-planner/src/test/java/sleeper/bulkexport/planner/SqsBulkExportProcessorLambdaIT.java

```diff
@@ -16,32 +16,108 @@
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static sleeper.core.properties.instance.CdkDefinedInstanceProperty.BULK_EXPORT_QUEUE_URL;
 import static sleeper.core.properties.instance.CdkDefinedInstanceProperty.CONFIG_BUCKET;
+import static sleeper.core.properties.instance.CdkDefinedInstanceProperty.LEAF_PARTITION_BULK_EXPORT_QUEUE_URL;
+import static sleeper.core.properties.table.TableProperty.TABLE_NAME;
 import static sleeper.core.properties.testutils.InstancePropertiesTestHelper.createTestInstanceProperties;
+import static sleeper.core.properties.testutils.TablePropertiesTestHelper.createTestTableProperties;
+import static sleeper.core.statestore.AllReferencesToAFileTestHelper.fileWithNoReferences;
+import static sleeper.core.statestore.AllReferencesToAFileTestHelper.fileWithReferences;
+import static sleeper.core.statestore.testutils.StateStoreUpdatesWrapper.update;
 
 public class SqsBulkExportProcessorLambdaIT extends LocalStackTestBase {
 
+    private static final Schema KEY_VALUE_SCHEMA = Schema.builder()
+            .rowKeyFields(new Field("key", new StringType()))
+            .valueFields(new Field("value1", new StringType()), new Field("value2", new StringType()))
+            .build();
+
     private InstanceProperties instanceProperties;
     private SqsBulkExportProcessorLambda sqsBulkExportProcessorLambda;
+    private TableProperties tableProperties;
+
+    @TempDir
+    public Path tempDir;
 
     @BeforeEach
-    public void setUp() throws ObjectFactoryException {
+    public void setUp() throws ObjectFactoryException, IOException {
         creatInstanceProperties();
+        tableProperties = createTestTableProperties(instanceProperties, KEY_VALUE_SCHEMA);
+        DynamoDBTableIndexCreator.create(dynamoClient, instanceProperties);
+        S3TableProperties.createStore(instanceProperties, s3Client, dynamoClient).save(tableProperties);
+        setupTransactionLogStateStore(tableProperties);
         sqsBulkExportProcessorLambda = new SqsBulkExportProcessorLambda(sqsClient, s3Client, dynamoClient, instanceProperties.get(CONFIG_BUCKET));
     }
 
+    @Test
+    public void testProcessorLambdaProducesLeafPartitionMessages() {
+        // Given
+        String tableName = tableProperties.get(TABLE_NAME);
+        SQSEvent event = createEvent("{\"tableName\":\"" + tableName + "\"}");
+
+        // When
+        sqsBulkExportProcessorLambda.handleRequest(event, null);
+
+        // Then
+        List<ReceiveMessageResult> results = new ArrayList<>();
+        ReceiveMessageResult result;
+        do {
+            result = sqsClient.receiveMessage(new ReceiveMessageRequest(instanceProperties.get(LEAF_PARTITION_BULK_EXPORT_QUEUE_URL)));
+            if (result.getMessages().size() > 0) {
+                results.add(result);
+            }
+        } while (result.getMessages().size() > 0);
+        assertThat(results).hasSize(2);
```

Comment:
You can set maxNumberOfMessages on the ReceiveMessageRequest, to a maximum of 10. This avoids the need for repeat calls.

You can also parse the messages from the queue with BulkExportLeafPartitionQuerySerDe and assert on the query objects, similar to SQSResultsOutputIT.